### PR TITLE
Order exported type for content export step as discussed in issue #2491

### PR DESCRIPTION
**Pull request by psijkof**
For issue #2491, ordering contenttype in content deployment step

### CODE REVIEW

**Comment by psijkof** [comment]
Also fixes issue #2478

File: .gitignore [hunk 1]
```diff
@@ -192,3 +192,4 @@ wwwroot
 !src/Templates/**/content/**
 .template.config/
 /OrchardCore.sln.GhostDoc.xml
+/src/Templates/OrchardCore.Cms.Templates/content
```
Comment (sebastienros) [inline .gitignore]: What is that ?

File: src/OrchardCore.Modules/OrchardCore.Settings/Recipes/SettingsStep.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 using System;
+using System.Linq;
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Settings/Recipes/SettingsStep.cs]: I think you can remove the `using`

File: .gitignore (hunk 1 above)
Comment (psijkof) [reply .gitignore]: It added some files under that folder, I guess not needed, so I ignored the files and gitignore was changed.

File: src/OrchardCore.Modules/OrchardCore.Settings/Recipes/SettingsStep.cs (hunk 2 above)
Comment (psijkof) [reply src/OrchardCore.Modules/OrchardCore.Settings/Recipes/SettingsStep.cs]: done.

File: .gitignore (hunk 1 above)
Comment (sebastienros) [reply .gitignore]: What is "it" ?
```suggestion
```

Comment (psijkof) [reply .gitignore]: Should I remove the line?

Comment (psijkof) [reply .gitignore]: Apparently VS generated bin and obj files in that location

**Pull request merged**